src/main/java/org/cardanofoundation/conversions/converters/SlotConversions.java

```diff
@@ -26,4 +27,14 @@ public LocalDateTime slotToTime(long absoluteSlot) {
     // for now post byron we have 1 slot = 1 second
     return genesisConfig.blockTime(EraType.Shelley, absoluteSlot);
   }
+
+    public Long slotToEpoch(long absoluteSlot) {
```

Comment:
Can we somehow indicate that this is partial function in Java?

https://www.pgrs.net/2015/04/23/partial-function-application-in-java-8/

like if this is byron then function doesn't work right..

What do you think?